images/kubectl-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -1696,20 +1696,22 @@ patchBuildStep "${buildStartTime}" "${previousStepEnd}" "${currentStepEnd}" "${N
 previousStepEnd=${currentStepEnd}
 set -x
 
-##############################################
-### RUN sbom generation and store in configmap
-##############################################
+if [ "$(featureFlag INSIGHTS_SBOM)" = enabled ]; then
```

Comment:
Let's just make it INSIGHTS - to save an overproliferation?

```suggestion
if [ "$(featureFlag INSIGHTS)" = enabled ]; then
```

Reply:
Sure, could there be other things though? I wasn't sure